.travis.yml

```diff
@@ -118,8 +118,24 @@ jobs:
 
                 # Composer: boost installation
                 - composer global show hirak/prestissimo -q || travis_retry composer global require $DEFAULT_COMPOSER_FLAGS hirak/prestissimo
+
+                # Require PHPUnit 8
+                - composer require --dev --no-update phpunit/phpunit:^8
+
+                # Install PCOV
+                - |
+                    git clone --branch=release --depth=1 https://github.com/krakjoe/pcov
+                    cd pcov
+                    phpize
```

Comment:
just wondering can we not build it each time, but maybe have prebuilt package we can use locally as well?

Reply:
I was afraid it would take long and be problematic, but it turned out to be fast - no more than 5 seconds.

What do you mean by prebuilt package? I imagine pcov will be official PHP extension, one day.

Reply:
like a php docker image with phpize already built.
so, one would be able to execute code coverage as well on his local machine, using same command to generate it, without need to manually modify his own php installation on his host

Reply:
Can we do it separately? I'm not sure my docker's knowledge is good enough.